x-pack/plugins/reporting/public/share_context_menu/register_pdf_png_reporting.tsx

```diff
@@ -38,6 +38,7 @@ const jobParamsProvider = ({
     browserTimezone,
     layout: sharingData.layout as LayoutParams,
     title: sharingData.title as string,
+    version: '7.14.0',
```

Comment:
I'm not really clear on why have this value here - I was expecting this to be dynamically set to the current Kibana version. At the moment this is the `UNVERSIONED_VERSION` - do we want that injected into the job params?

Reply:
Oops! This was left over from an earlier iteration, and it is wrong.